### PR TITLE
Add 3 web applets

### PR DESCRIPTION
**Pull request by MariaNattestad**
- Add 3 example web applications: Dash (python), R Shiny, and TensorBoard.
- Add a web app section to the tutorials page with these 3 web applets
- Put the link to the source on GitHub prominently at the top of each applet page, and make it open in a new tab
- Minor additions to notes on contributing and running it locally

### CODE REVIEW

File: Tutorials/bash/tensorboard-web-app/resources/home/dnanexus/mnist_tensorboard_example.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+This is an unimpressive MNIST model, but it is a good example of using
+tf.name_scope to make a graph legible in the TensorBoard graph explorer, and of
+naming summary tags so that they are grouped meaningfully in TensorBoard.
+
+It demonstrates the functionality of every TensorBoard dashboard.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```
Comment (Damien-Black) [inline Tutorials/bash/tensorboard-web-app/resources/home/dnanexus/mnist_tensorboard_example.py]: +1 for python3 compatibility. Did `from __future__ import unicode_literals` cause issues?

Comment (MariaNattestad) [reply Tutorials/bash/tensorboard-web-app/resources/home/dnanexus/mnist_tensorboard_example.py]: nope, everything just worked! This code is straight from the TensorBoard examples (hence the license at the top), so I'm not gonna take credit for this :)

Comment (bhuffman) [reply Tutorials/bash/tensorboard-web-app/resources/home/dnanexus/mnist_tensorboard_example.py]: I really think we should have gone for 4 examples here. Come on, guys...

Comment (MariaNattestad) [reply Tutorials/bash/tensorboard-web-app/resources/home/dnanexus/mnist_tensorboard_example.py]: lol, okay I'll have to get to work on that :)

**Pull request merged**